man/systemd.network.xml

```diff
@@ -960,6 +960,14 @@
         </varlistentry>
 
         <varlistentry>
+          <term><varname>UserClass=</varname></term>
+          <listitem>
+            <para>The user class used define specific user class identifiers to convey information
+            about a client's software configuration or about its user's preferences.</para>
+          </listitem>
+        </varlistentry>
```

Comment:
as far as I can see multiple user classes may be specified within the same dhcp datagram. I figure this should be reflected by the option: it should be usable multiple times, each adding one additional class.